docs/zh-CN/sql-reference/sql-functions/date-time-functions/curdate.md

```diff
@@ -24,7 +24,7 @@ specific language governing permissions and limitations
 under the License.
 -->
 
-# curdate
+# curdate,`current_date`
```

Comment:
```suggestion
# curdate,current_date
```

Reply:
Thanks for reminding. It has been modified